crates/service_utils/Cargo.toml

```diff
@@ -35,6 +35,7 @@ reqwest = { workspace = true }
 thiserror = { workspace = true }
 once_cell = { workspace = true }
 regex = { workspace = true }
+mime = "0.3.17"
```

Comment:
Can we inherit the version from the workspace - so that versions are same everywhere?